13-foundations-mathematical.Rmd

```diff
@@ -25,6 +25,17 @@ While they differ in the settings, in their outcomes, and also in the technique
 terms_chp_13 <- c("sampling distribution", "null distribution")
 ```
 
+::: {.important data-latex=""}
+**Sampling distribution.**
+
+A sampling distribution describes the possible values of a **statistic** (note that here the sample proportion, $\hat{p}$ is described, but other statistics like the sample mean, $\bar{x}$ will be described later) as they change from study to study.\
+A sampling distribution is contrasted with a data distribution which shows the variability of the *observed* data values.
+
+A data distribution can be visualized from the observations themselves.
+However, because a sampling distribution describes many statistics computed from many studies, it cannot be visualized directly from a single dataset.
+Instead, a sampling distribution uses either computational or mathematical structures to describe the variability expected in repeated studies.
```

Comment:
```suggestion
A sampling distribution is contrasted with a data distribution which shows the variability of the *observed* data values.
The data distribution can be visualized from the observations themselves.
However, because a sampling distribution describes sample statistics computed from many studies, it cannot be visualized directly from a single dataset.
Instead, we use either computational or mathematical structures to estimate the sampling distribution and hence to describe the expected variability of the sample statistic in repeated studies.
```